code/modules/surgery/surgery_step.dm

```diff
@@ -175,4 +175,4 @@
 /datum/surgery_step/proc/display_results(mob/user, mob/living/carbon/target, self_message, detailed_message, vague_message, target_detailed = FALSE)
 	user.visible_message(detailed_message, self_message, vision_distance = 1, ignored_mobs = target_detailed ? null : target)
 	if(!target_detailed)
-		to_chat(target, vague_message)
+		target.show_message(vague_message, MSG_VISUAL)
```

Comment:
Mentioned it on Discord but I'll double it up here.

Can ya add a cheeky little alt_msg to show_message, something generic that lets blind people know that some surgery procedure is happening to them and it is causing them pain.

Once that is done, this is all good to merge in my opinion!

Reply:
Done.